src/profile/components/MainMenu.tsx

```diff
@@ -1,38 +1,38 @@
 import classnames from 'classnames';
+import { useRouter } from 'next/router';
+import React from 'react';
+
+import { getMyProfileUrl, getProfileSearchUrl, getProfileSettingsUrl, getProfileStatisticsUrl } from 'core/appUrls';
 import { Link } from 'core/components/Router';
-import React, { Component } from 'react';
-import Router from 'react-router-dom';
-import { routes } from '../index';
 import style from './mainMenu.less';
 
-export interface IProps {
-  match: Router.match<{}>;
-}
-
-class MainMenu extends Component<IProps, {}> {
-  public render() {
-    const { path } = this.props.match;

```

Comment:
Did you intentionally remove the feature of being able to see what menu element is the current chosen one, is it a mistake or is it being fixed somewhere else?

Reply:
I Moved the feature down a few levels, into the component which displays the url. Before hooks we had to get the `props.match.path` directly from the child of a `Route` component. With hooks the path is part of a context, which we can consume anywhere we need it in the tree